src/index.css

```diff
@@ -9,6 +9,7 @@ html{
   width: 100%;
   max-width: 100vw;
   overflow-x: hidden;
+  /* overflow-y: hidden; */
```

Comment:
Just a warning that this was fixed in one of the prs so you may need to merge in the latest develop branch and fix some conflicts before we can merge